src/shell/Shell.cpp

```diff
@@ -872,7 +902,7 @@ static void executeWAST(Store* store, const std::string& filename, const std::ve
     }
 }
 
-static void runExports(Store* store, const std::string& filename, const std::vector<uint8_t>& src, std::string& exportToRun)
+static void runExports(Store* store, const std::string& filename, const std::vector<uint8_t>& src, std::string& exportToRun, SpecTestFunctionTypes& functionTypes)
```

Comment:
AFAIK, `runExports` method is used only for test code, especially compiled from c to wasm by `emcc` 
If we support the `_start` function, then this `runExports` is no longer necessary I think.
What do you think about this?

Reply:
It could be left in for test purposes and be moved into another environment that is only used in debug mode if we really dont need it. That is what some other runtimes do as I looked into them.

Reply:
I see, let's keep this part as is.